src/app/js/admin/parsing/ParsingExcerpt.js

```diff
@@ -69,10 +90,16 @@ export const ParsingExcerptComponent = ({
                                 showAddColumns &&
                                 (index === total - 3 ||
                                     (total < 3 && index === 0));
+
                             return (
                                 <ParsingExcerptColumn
                                     key={`${column}_${line._id}`}
                                     value={`${line[column] || ''}`}
+                                    style={
+                                        arrayIsColumnEnrichment[column] && {
```

Comment:
I think you'd better add an entry in the `styles` const, and reference it in here.